### PR TITLE
Fixes issue:8631: Update the steps of test case to get rgw_alerts

### PR DESCRIPTION
**Pull request by Akarsha-rai**
Fixes: #8631

### CODE REVIEW

File: tests/functional/monitoring/conftest.py [hunk 1]
```diff
@@ -905,8 +905,17 @@ def stop_rgw():
         nonlocal rgw_deployments
         for rgw_deployment in rgw_deployments:
             rgw = rgw_deployment["metadata"]["name"]
-            logger.info(f"Downscaling deployment {rgw} to 0")
-            oc.exec_oc_cmd(f"scale --replicas=0 deployment/{rgw}")
+            logger.info(
+                f"Updating readiness probe of deployment {rgw} to make rgw pod down"
+            )
+            cmd = (
```
Comment (fbalak) [inline tests/functional/monitoring/conftest.py:911]: The change in readiness probe should be restored when testing is done.

**Review by fbalak — CHANGES_REQUESTED** [review]

File: tests/functional/monitoring/conftest.py (hunk 1 above)
Comment (Akarsha-rai) [reply tests/functional/monitoring/conftest.py:911]: At the end deleting the deployment rgw and operator pod restarts it with the right readiness probe condition. This has been verified.

**Review by fbalak — DISMISSED** [review]

File: tests/functional/monitoring/prometheus/alerts/test_rgw.py [hunk 2]
```diff
@@ -38,7 +40,10 @@ def test_rgw_unavailable(measure_stop_rgw, threading_lock):
             "Please check Ceph cluster health or RGW connection."
         )
     else:
-        target_msg = "Cluster Object Store is in unhealthy state. Please check Ceph cluster health."
+        target_msg = (
+            "Cluster Object Store is in unhealthy state or number of ready replicas for "
+            "Rook Ceph RGW deployments is less than the desired replicas in namespace:cluster openshift-storage:."
```
Comment (DanielOsypenko) [inline tests/functional/monitoring/prometheus/alerts/test_rgw.py:45]: `openshift-storage` should not be hardcoded. Lets pick it up from config.ENV_DATA

Comment (Akarsha-rai) [reply tests/functional/monitoring/prometheus/alerts/test_rgw.py]: ack, used `constants.OPENSHIFT_STORAGE_NAMESPACE`

File: tests/functional/monitoring/prometheus/alerts/test_rgw.py [hunk 3]
```diff
@@ -38,7 +40,11 @@ def test_rgw_unavailable(measure_stop_rgw, threading_lock):
             "Please check Ceph cluster health or RGW connection."
         )
     else:
-        target_msg = "Cluster Object Store is in unhealthy state. Please check Ceph cluster health."
+        target_msg = (
+            "Cluster Object Store is in unhealthy state or number of ready replicas for "
+            "Rook Ceph RGW deployments is less than the desired replicas in "
+            f"namespace:cluster {constants.OPENSHIFT_STORAGE_NAMESPACE}:."
```
Comment (DanielOsypenko) [inline tests/functional/monitoring/prometheus/alerts/test_rgw.py:46]: pls use `config.ENV_DATA["cluster_namespace"]`

Comment (Akarsha-rai) [reply tests/functional/monitoring/prometheus/alerts/test_rgw.py]: done

**Review by DanielOsypenko — APPROVED** [review]

**Review by vavuthu — APPROVED** [review]

**Pull request merged**